tests/testthat/test-bind.R

```diff
@@ -378,9 +391,18 @@ test_that("performance: Row binding with df-cols doesn't duplicate on every assi
 # cols --------------------------------------------------------------------
 
 test_that("vec_cbind() reports error context", {
-  expect_snapshot({
-    (expect_error(vec_cbind(foobar(list()))))
-    (expect_error(vec_cbind(a = 1:2, b = int())))
+  expect_snapshot(error = TRUE, {
```

Comment:
I think in a package like vctrs we'll still want to monitor error classes. With this change, the class no longer appears in snapshots. Can you revert to the previous pattern and also use it in the other tests please?

Reply:
Sounds good, I've changed back to the previous pattern and re-requested a review